vignettes/baselinenowcast.Rmd

```diff
@@ -82,25 +82,17 @@ For this quick start, we use daily level data from the
 These data represent hospitalisation counts by date of positive test and date
 of test report in Germany up to October 1, 2021.
 
-# Filtering
+
+# Filtering and plotting the data
 
 We will filter the data to just look at the national-level data, for all age
 groups.
 We will pretend that we are making a nowcast as of July 1, 2021, therefore we
 will exclude all reference dates and report dates after that date.
 `germany_covid19_hosp` is provided as package data from `epinowcast`
-```{r load-data}
-observed_long <- epinowcast::germany_covid19_hosp |>
```

Comment:
as above I dont think you need to use package function here for the epinowcast data as you already say where it is from. I would actually rename it on first talking about it to something generic to make it easier for people to copy and paste this code and use it on their own data.